Tasks/count.js

```diff
@@ -1,17 +1,12 @@
-// Sum all number values in dict
+'use strict'
 
-count = (obj) => {
-  sum = 0;
-  {
+ const adder = (obj) => {
+  let sum = 0;
+  const arrValues = Object.values(obj);
+  for (const element of arr_of_values) {
```

Comment:
This should not work, `arr_of_values` is not defined
```suggestion
  for (const element of arr_of_values) {
```

Reply:
'use strict';

 const adder = (obj) => {
  let sum = 0;
  const arr_of_values = Object.values(obj);
  for (const element of arr_of_values) {
    if (typeof element === 'number') sum +=   element;
  }
  return sum;
};

require('../Tests/count.js')(adder);

Reply:
correct variant